rules/valid-describe.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const describeAliases = Object.assign(Object.create(null), {
```

Comment:
This was taken from `no-identical-title`:

https://github.com/jest-community/eslint-plugin-jest/blob/5e5ceba70f13ed2a909744adde82a6ed15fe4768/rules/no_identical_title.js#L3-L9

I can extract an `isDescribe(node)` function, move it to `util.js` and import it into `no-identical-title` and this rule, if that refactor would be helpful.

Reply:
yeah, that sounds good